zipkin-autoconfigure/storage-elasticsearch/src/main/resources/META-INF/spring.factories

```diff
@@ -1,4 +1,4 @@
 org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
-zipkin.autoconfigure.storage.elasticsearch.http.ZipkinElasticsearchOkHttpAutoConfiguration,\
-zipkin.autoconfigure.storage.elasticsearch.http.ZipkinElasticsearchStorageAutoConfiguration,\
-zipkin.autoconfigure.storage.elasticsearch.http.TracingZipkinElasticsearchStorageAutoConfiguration
+zipkin2.autoconfigure.storage.elasticsearch.ZipkinElasticsearchOkHttpAutoConfiguration,\
+zipkin2.autoconfigure.storage.elasticsearch.ZipkinElasticsearchStorageAutoConfiguration,\
```

Comment:
this one is public, but I have no problem creating a change in zipkin-aws to accomodate